src/test/java/seedu/duke/command/EditTransactionCommandTest.java

```diff
@@ -0,0 +1,115 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.classes.StateManager;
+import seedu.duke.exception.DukeException;
+import seedu.duke.parser.Parser;
+import seedu.duke.ui.Ui;
+
+import java.io.ByteArrayOutputStream;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class EditTransactionCommandTest {
+
+    private static Parser parser = new Parser();
+    private static ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+    private static Ui ui = new Ui(outputStream);
+
+    @BeforeEach
+    void populateStateManager() {
+        try {
```

Comment:
You are missing out on the goal commands
![image](https://github.com/AY2324S1-CS2113-W12-3/tp/assets/33816512/cd1a6027-740c-47a7-ae0c-7936a7f85258)